src/lib/viewers/BaseViewer.js

```diff
@@ -1055,10 +1055,13 @@ class BaseViewer extends EventEmitter {
      * Handler for annotation toolbar region comment button click event.
      *
      * @private
+     * @param {boolean} activeControl - avtive control mode in AnnotationControls
      * @return {void}
      */
-    handleRegionClick() {
-        this.annotator.toggleAnnotationMode(AnnotationMode.REGION);
+    handleRegionClick({ activeControl }) {
+        if (activeControl === AnnotationMode.REGION) {
+            this.annotator.toggleAnnotationMode(AnnotationMode.REGION);
```

Comment:
Would we need to explicitly set it back to AnnotationMode.NONE?

Reply:
It's already set to `AnnotationMode.NONE` in `resetControls`.